doc/modules/changes/20190530_jeanniot

```diff
@@ -0,0 +1,4 @@
+New: Extended spherical shell geometry model to 
+include custom mesh schemes. 
+<br>
+(Ludovic Jeanniot and Marie Kajan, 2019/05/30)
```

Comment:
This really only lists 2 of the 3 authors of #3097 ;-) ;-)

Reply:
That is not cool from me to have not listed the most important author of this new feature. 
My apologies for this oversight, I just opened a pull request to rectify the changes log #3143 (related to #3097)  :)